plugin.json

```diff
@@ -2,22 +2,25 @@
     "name": "VisitorGenerator",
     "version": "3.0.1",
     "description": "Developer tool that lets you generate fake visits. Useful if you are working with plugins or themes or if you use the Piwik API.",
-    "keywords": ["development", "tools"],
+    "keywords": [
+        "development",
+        "tools"
+    ],
     "license": "GPL v3+",
-    "homepage": "http://piwik.org",
+    "homepage": "http:\/\/piwik.org",
     "require": {
-        "piwik": ">=3.0.0-b1,<4.0.0-b1"
+        "piwik": ">=3.3.1-b4,<4.0.0-b1"
```

Comment:
I think this version requirement is not needed? (The command generator has likely updated this automatically since we don't expect plugin developers to test it with older Matomo versions)